watertap/property_models/anaerobic_digestion/adm1_reactions.py

```diff
@@ -1216,7 +1216,7 @@ def _rxn_rate(self):
         self.reaction_rate = pyo.Var(
             self.params.rate_reaction_idx,
             initialize=self.rates,
-            bounds=(1e-9, 1e-4),
+            bounds=(1e-10, 1e-4),
```

Comment:
I might suggest we modify these bounds similar to what I did in the PR on modified ADM1 (#948)

Reply:
Agree, however I think we should take a closer look after the thickener unit model is added to see the effects on concentration to understand where the bounds would look like